broker/broker-security-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.2.0">
+    <OCD name="Subject Injector Plugin Configuration"
+         id="org.codice.ddf.broker.security.impl.SubjectInjectorPlugin">
+
+        <AD
+                description="The addresses that should have the Subject injected into the messages"
```

Comment:
❓ Not a blocker, but is there a plan to update docs with this configuration?

Reply:
I think there should be. I could make a ticket

Reply:
This will get tracked on: https://codice.atlassian.net/browse/DDF-3618